client/browser/code-intel-extensions.json

```diff
@@ -1,40 +1 @@
-[
-  "sourcegraph/apex",
-  "sourcegraph/clojure",
-  "sourcegraph/cobol",
-  "sourcegraph/cpp",
-  "sourcegraph/csharp",
-  "sourcegraph/cuda",
-  "sourcegraph/dart",
-  "sourcegraph/elixir",
-  "sourcegraph/erlang",
-  "sourcegraph/go",
-  "sourcegraph/graphql",
-  "sourcegraph/groovy",
-  "sourcegraph/haskell",
-  "sourcegraph/java",
-  "sourcegraph/jsonnet",
-  "sourcegraph/kotlin",
-  "sourcegraph/lisp",
-  "sourcegraph/lua",
-  "sourcegraph/ocaml",
-  "sourcegraph/pascal",
-  "sourcegraph/perl",
-  "sourcegraph/php",
-  "sourcegraph/powershell",
-  "sourcegraph/protobuf",
-  "sourcegraph/python",
-  "sourcegraph/r",
-  "sourcegraph/ruby",
-  "sourcegraph/rust",
-  "sourcegraph/scala",
-  "sourcegraph/shell",
-  "sourcegraph/starlark",
-  "sourcegraph/strato",
-  "sourcegraph/swift",
-  "sourcegraph/tcl",
-  "sourcegraph/thrift",
-  "sourcegraph/typescript",
-  "sourcegraph/verilog",
-  "sourcegraph/vhdl"
-]
+["sourcegraph/apex","sourcegraph/clojure","sourcegraph/cobol","sourcegraph/cpp","sourcegraph/csharp","sourcegraph/cuda","sourcegraph/dart","sourcegraph/elixir","sourcegraph/erlang","sourcegraph/go","sourcegraph/graphql","sourcegraph/groovy","sourcegraph/haskell","sourcegraph/java","sourcegraph/jsonnet","sourcegraph/kotlin","sourcegraph/lisp","sourcegraph/lua","sourcegraph/ocaml","sourcegraph/pascal","sourcegraph/perl","sourcegraph/php","sourcegraph/powershell","sourcegraph/protobuf","sourcegraph/python","sourcegraph/r","sourcegraph/ruby","sourcegraph/rust","sourcegraph/scala","sourcegraph/shell","sourcegraph/starlark","sourcegraph/strato","sourcegraph/swift","sourcegraph/tcl","sourcegraph/thrift","sourcegraph/typescript","sourcegraph/verilog","sourcegraph/vhdl"]
```

Comment:
Newline? Was this autoformatted by them or us?

Reply:
I have no idea what this is 🤔

Reply:
Guess we should revert it?